radixdlt-java/src/main/java/com/radixdlt/client/application/translate/ApplicationStore.java

```diff
@@ -24,8 +25,7 @@ public ApplicationStore(
 			particleStore.getParticles(address)
 				.scanWith(reducer::initialState, reducer::reduce)
 				.debounce(1000, TimeUnit.MILLISECONDS)
-				.replay(1)
-				.autoConnect()
+				.compose(ReplayingShare.instance())
```

Comment:
Do we really need to include an entire library for this line?

Reply:
Yes, the whole library is actually just this one line. It's a super useful construct which can't be replicated easily in RXJava.